trace_decoder/src/processed_block_trace.rs

```diff
@@ -150,6 +150,13 @@ impl TxnInfo {
                     }
                 }
             }
+
+            if trace
+                .self_destructed
+                .map_or(false, |self_destructed| self_destructed)
```

Comment:
nit: isn't this `Option::unwrap_or_default`?

Reply:
yeah it is, just brought back old code :) I changed it